staging/src/k8s.io/apiserver/pkg/storage/storagebackend/config.go

```diff
@@ -49,6 +49,7 @@ type Config struct {
 
 func NewDefaultConfig(prefix string, copier runtime.ObjectCopier, codec runtime.Codec) *Config {
 	return &Config{
+		Type:   StorageTypeETCD3,
```

Comment:
Then I'd prefer it here. Otherwise validation and user facing info has to "just know" what empty means (and we have to keep it in sync manually)